### PR TITLE
Make use of provided logger functions

### PR DESCRIPTION
**Pull request by Nocccer**
To make the output to console or log file more readable, i replaced all console.log and console.error with the provided logger functions. 

I also made a function which checks commands if they fullfil the given version.
This should fix problems, where python version was not correctly parsed (e.g. "Python 3.8.2rc2")

checkCommancVersion now can parse version correctly for:
- "Command 1.0.0"
- "1.0.0 Command"
- "Command 1.0.0rc2"
- "1.0.0rc2 Command"

- [x] replace console.log and console.error with logger functions
- [x] create function which checks if commands fullfil version
- [x] make regex which parses Version out of a string

New console log:
```none
WARNING: Protocol already registered.
WARNING: python command not found
WARNING: python3 command not found
ERROR: Heroic requires Python 3.8 or newer.
INFO: launching Escolar
WARNING: GameMode not installed
INFO: 
 Launch Command:        /tmp/.mount_HeroicH9c0H8/resources/app.asar.unpacked/build/bin/linux/legendary launch Escolar  --wine /usr/bin/wine --wine-prefix '/home/niklas/.wine' 
INFO: Stopping Discord Rich Presence if running...
INFO: Stopped Discord Rich Presence.
```

----------------------------------

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Created / Updated Tests
- [ ] Created / Updated documentation

Tested on a Clean Install with AppImage:
- [x] Login
- [ ] Download a Game
- [ ] Import a Game
- [ ] Launch Default config
- [ ] Launch Personalized Config
- [x] Launch wine
- [ ] Launch Proton
- [ ] Launch Custom Proton
- [x] Launch Tools (Winetricks and Winecfg)
- [ ] Uninstall Game
- [ ] Move Game
- [x] Languages (Test if no translation stopped working)

### CODE REVIEW

File: electron/main.ts
Comment (Nocccer) [inline electron/main.ts:28]: remove

**Comment by dragonDScript** [comment]
👏

**Review by flavioislima — DISMISSED** [review]
Looks nice 👍

**Review by dragonDScript — APPROVED** [review]
LGTM

**Comment by Nocccer** [comment]
@flavioislima 
Something after the merge broke the frontend. Test's failing but i didn't touch that?

**Comment by flavioislima** [comment]
> @flavioislima
> Your changes here broke frontend tests.
> [9f0de70](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/commit/9f0de700c7cba010b349050a631c777461f18ac3)

Yes, sorry for that. 
Just wait until I merge #480, I fixed there.

**Comment by flavioislima** [comment]
@Nocccer just update the branch and the tests will be fixed

**Comment by Nocccer** [comment]
@flavioislima thanks. If you approve it, i can merge it.

**Review by flavioislima — APPROVED** [review]

**Pull request merged**
